frontend/testing/cypress/config/dev.json

```diff
@@ -2,14 +2,12 @@
   "$schema": "https://on.cypress.io/cypress.schema.json",
   "baseUrl": "https://dev.altinn.studio",
   "env": {
-    "autoTestUser": "AutoTest",
-    "useCaseUser": "bruksmonsterDev",
-    "datamodelApp": "AutoTest/datamodel",
-    "deployApp": "ttd/auto-deploy-app-v3",
-    "designerApp": "AutoTest/auto-designer-app",
-    "withoutDataModelApp": "AutoTest/appwithout-dm",
-    "appOwner": "Testdepartementet",
-    "appOwnerUsername": "ttd"
+    "autoTestUser": "dev_cypress_testuser",
+    "useCaseUser": "dev_bruksmonster",
```

Comment:
Har disse brukerne blitt opprettet i miljøet? Det er en del som må gjøres for å kunne faktisk bytte bruker her:
- bruker må opprettes i miljøet
- Bruker må legges til i riktig org (ttd)
- Token må opprettes og legges inn i relevante pipelines som secret
- Passord (og token) må legges inn i enten altinnpedia eller KV
Om dette ikke er gjort så vil jeg anbefale å gå tilbake til de gamle brukerne. Tenker at det viktigste her er at vi bruker samme _variabel_ for brukerne, om brukernavn varierer så kan vi leve med det.

Reply:
Ja regnet med at det var noe slikt. Da er det kanskje like greit å bare beholde de som de var?